public/app/core/components/NavBar/NavBar.tsx

```diff
@@ -1,15 +1,33 @@
-import React, { FC, useState } from 'react';
-import { useLocation } from 'react-router-dom';
-import { useDispatch, useSelector } from 'react-redux';
 import { css, cx } from '@emotion/css';
-import { cloneDeep } from 'lodash';
 import { GrafanaTheme2, NavModelItem, NavSection } from '@grafana/data';
-import { Icon, IconName, useTheme2 } from '@grafana/ui';
 import { locationService } from '@grafana/runtime';
+import { Icon, IconName, useTheme2 } from '@grafana/ui';
+import { updateNavIndex } from 'app/core/actions';
 import { Branding } from 'app/core/components/Branding/Branding';
 import config from 'app/core/config';
+import { getPerconaSettings, getPerconaUser } from 'app/percona/shared/core/selectors';
 import { KioskMode } from 'app/types';
-import { updateNavIndex } from 'app/core/actions';
+import { cloneDeep } from 'lodash';
+import React, { FC, useState } from 'react';
```

Comment:
What's the reason for these order changes in imports?

Reply:
I added auto organize imports on save in vscode so it can remove unused imports on save and also organized them. I can turn it off if it is problem.

Reply:
Hm, it's bit weird, because importing React usually comes at the top, but your editor is putting way down

Reply:
Yeah, you are right, that is weird. I didn't noticed it. Unfortunately there is no setting for it in vscode so I disabled it and move it back to the top.